docs/src/main/sphinx/object-storage/metastores.md

```diff
@@ -491,6 +491,9 @@ following properties:
   - The credential to exchange for a token in the OAuth2 client credentials flow
     with the server. A `token` or `credential` is required for `OAUTH2`
     security. Example: `AbCdEf123456`
+* - `iceberg.rest-catalog.prefix`
```

Comment:
nit: I would move to under `iceberg.rest-catalog.uri`.

Reply:
done